x-pack/plugins/cases/public/components/all_cases/selector_modal/all_cases_selector_modal.test.tsx

```diff
@@ -82,13 +82,4 @@ describe('AllCasesSelectorModal', () => {
 
     expect(res.queryByTestId('cases-metrics-stats')).toBeFalsy();
   });
-
-  it('should show the create case button', async () => {
-    const res = appMockRenderer.render(<AllCasesSelectorModal {...defaultProps} />);
-    await waitFor(() => {
-      expect(res.getByTestId('cases-table')).toBeInTheDocument();
-    });
-
-    expect(res.getByTestId('cases-table-add-case-filter-bar')).toBeInTheDocument();
-  });
 });
```

Comment:
If which columns are displayed is now specific to the selector modal, I think we could now add a test to guarantee they show up.